reputation/peer.go

```diff
@@ -27,7 +26,7 @@ func PeerIDFromV2(id *reputation.PeerID) *PeerID {
 }
 
 // SetPublicKey sets peer ID as a compressed public key.
-func (x *PeerID) SetPublicKey(v [signature.PublicKeyCompressedSize]byte) {
+func (x *PeerID) SetPublicKey(v [33]byte) {
```

Comment:
Magic constant, worth defining in `crypto/ecdsa`.

Reply:
Argument of this method is going to be changed to `[]byte` in release.

Reply:
Is there a task/PR for that?